tools/shared.py

```diff
@@ -444,10 +444,11 @@ def normalize_config_settings():
     PORTS = os.path.join(CACHE, 'ports')
 
   if CLOSURE_COMPILER is None:
-    CLOSURE_COMPILER = get_npm_cmd('google-closure-compiler')
-    if not WINDOWS:
+    if WINDOWS:
+      CLOSURE_COMPILER = [path_from_root('node_modules', '.bin', 'google-closure-compiler.cmd')]
+    else:
       # Work around an issue that Closure compiler can take up a lot of memory and crash in an error
-      # "FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory"
+     # "FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory"
```

Comment:
indentation looks wrong?

Reply:
The indentation is wrong, but that is how it used to be before #12139

I'm about to do a followup so I'm not concerned about it.